drivers/serial/uart_nrfx_uarte.c

```diff
@@ -182,6 +184,31 @@ struct uarte_nrfx_data {
  */
 #define UARTE_CFG_FLAG_LOW_POWER   BIT(4)
 
+/* Macro for converting numerical baudrate to register value. It is convenient
+ * to use this approach because for constant input it can calculate nrf setting
+ * at compile time.
+ */
+#define NRF_BAUDRATE(baudrate) ((baudrate) == 300 ? 0x00014000 :\
```

Comment:
Wouldn't it be good to use this macro also in `baudrate_set()` so that the translation is done in one place only?

Reply:
changed `baudrate_set()` to use `NRF_BAUDRATE()`